pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/github_scraper.go

```diff
@@ -228,13 +228,13 @@ func (ghs *githubScraper) scrape(ctx context.Context) (pmetric.Metrics, error) {
 				comCount := 100
 
 				var cc *string
-
+				var totalAdditions int = 0
```

Comment:
note: if you accept the two below suggestions then you'll have to fix the other area too (i'll add suggestions for it). These suggestions are for brevity